config/jobs/kubernetes-sigs/kubebuilder/kubebuilder-presubmits.yaml

```diff
@@ -1,231 +1,239 @@
 presubmits:
   kubernetes-sigs/kubebuilder:
-  - name: pull-kubebuilder-test
```

Comment:
/hold

We are removing and changing the test
The diff needs to be only what we are proposing to change here.